test/helpers.test.js

```diff
@@ -24,5 +24,11 @@ describe('HELPER TEST SUITE', () => {
       expect(result[0]).to.equal('Science');
       done();
     });
+
+    it('Should not throw up a TypeError when no input is supplied', (done) => {
+      const result = convertToArray();
+      expect(result);
```

Comment:
I think it's best to expect result to be something specifically. This line is like saying `true`. And since it's true, it will always pass. We need to see something done with `result`, maybe the type or the value of the elements in it. Anything apart from `expect(result)`

Reply:
Thanks, I've made the change